src/Engine/Data/RenderParameters.hpp

```diff
@@ -218,7 +363,23 @@ class RA_ENGINE_API RenderParameters final
     UniformBindableSet<Mat4Parameter> m_mat4ParamsVector;
 
     UniformBindableSet<TextureParameter> m_texParamsVector;
-    /**@}*/
+    /**\}*/
+};
+
+/**
+ * Interface to define metadata (constraints, description, ...) for the edition of parameter set
+ */
+class RA_ENGINE_API ParameterSetEditionInterface
```

Comment:
```suggestion
class RA_ENGINE_API ParameterSetEditingInterface
```
As far as I know, edition is related to printing, editing is related to the edit action.